templates/head/google_fonts.twig

```diff
@@ -1,45 +1,46 @@
 <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700,900&display=swap&subset=latin-ext"/>
 <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lora:400,400i,700&display=swap&subset=latin-ext"/>
 {% if  custom_font_families %}
+	{% set fonts_dir = theme_uri ~ "/../../plugins/planet4-plugin-gutenberg-blocks/public/fonts" %}
```

Comment:
Just to make it work again. Still need to investigate if the old code worked previously and why it broke.

Reply:
Fixed in https://github.com/greenpeace/planet4-master-theme/pull/1427